_content/av/early-buddhist-tok-course_sujato.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Early Buddhist Theory of Knowledge"
+authors:
+  - sujato
+subcat: course
+external_url: "https://www.youtube.com/@BuddhistLibrarySydney/search?query=theory%20of%20knowledge"
```

Comment:
Clever!  I also couldn't find a good playlist link, so I emailed the Sydney Buddhist Library to ask them to make a playlist for the course. We'll see if/how they respond. Worst case we can make a playlist ourselves I think 😊

Reply:
Okay. It's been a week and they still haven't replied. Here's a playlist I made for it: https://youtube.com/playlist?list=PLKLA1tWul8wGopojxI0vigUp1coMoIJe-